ui/src/app/base/components/LabelledList/_index.scss

```diff
@@ -5,7 +5,7 @@
     .p-list__item {
       @extend %vf-list-item;
       display: flex;
-      padding: $spv-inner--x-small--scaleable 0;
+      padding: $spv-inner--x-small--scaleable 0 !important;
```

Comment:
Vanilla lists have a different padding inside forms, but since this is a custom component we need to make sure it doesn't change.

Reply:
I reckon that would make a good comment in the code.